### PR TITLE
Relation

### LINKED ISSUE #346: Add a diagram to the OAM spec
@artursouza Proposed to add a diagram to the spec. I think it's a good idea. Here is my version of the diagram that tries to describe the relations between various OAM resources. 

OAM types

### PR DESCRIPTION
**Pull request by anubhavsingh16**
In this PR, I'm adding wireframe of Relation btw various OAM resources in Readme through a link in the Introduction section.
and updates as suggested.

Fixes #346

### CODE REVIEW

File: 2.overview_and_terminology.md [hunk 1]
```diff
@@ -306,3 +306,6 @@ This form is not accepted as an alternative for the fully qualified version. It
 | Previous Part        | Next Part          |
 | ------------- |-------------|
 |[1. Purpose And Goals](1.purpose_and_goals.md)| [3. Workload Definition](3.workload.md)|
+
+
+[Relations between various OAM resources](/design/20200727-spec-v0.2.0-data-model.md)
```
Comment (resouer) [inline 2.overview_and_terminology.md:311]: Please put it at the end of this section: https://github.com/oam-dev/spec/blob/d9e003673696db682974d9e1b3fb9169336460bc/2.overview_and_terminology.md#overview-of-the-model, i.e. `Overview of the Model` section

File: README.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ roles responsible for building and running apps and operating infrastructure.
 
 For more details and user stories, see [introduction.md](./introduction.md).
 
+[Relations between various OAM resources](./assets/relations_between_various_OAM_resources.png)
```
Comment (resouer) [inline README.md:46]: We don't need this README now, just keep it in 2.overview_and_terminology.md

File: 2.overview_and_terminology.md (hunk 1 above)
Comment (anubhavsingh16) [reply 2.overview_and_terminology.md:311]: @resouer At the end of `Overview of the Model`.
alisoc

Comment (anubhavsingh16) [reply 2.overview_and_terminology.md:311]: at green arrow?

File: README.md (hunk 2 above)
Comment (anubhavsingh16) [reply README.md:46]: Ok

File: 2.overview_and_terminology.md (hunk 1 above)
Comment (resouer) [reply 2.overview_and_terminology.md:311]: Yes! Sorry that I missed the notification of this PR.

**Comment by resouer** [comment]
The other PR has been merged and the diagram is already included, I will close this one.

**Pull request closed without merging**